### ISSUE TITLE
TypeError: path must be a string

### ISSUE THREAD

**Issue by janl** [robertkowalski/nmo#1]
Latest `nmo` via `npm`.
```
> nmo -h
ERR! path must be a string 
ERR! TypeError: path must be a string
ERR!     at Object.fs.open (fs.js:430:11)
ERR!     at Object.fs.readFile (fs.js:207:6)
ERR!     at ConfigChain.addFile (/usr/local/lib/node_modules/nmo/node_modules/config-chain/index.js:196:6)
ERR!     at /usr/local/lib/node_modules/nmo/lib/config.js:85:48
ERR!     at tryCatcher (/usr/local/lib/node_modules/nmo/node_modules/bluebird/js/main/util.js:24:31)
ERR!     at Promise._resolveFromResolver (/usr/local/lib/node_modules/nmo/node_modules/bluebird/js/main/promise.js:427:31)
ERR!     at new Promise (/usr/local/lib/node_modules/nmo/node_modules/bluebird/js/main/promise.js:53:37)
ERR!     at Object.load (/usr/local/lib/node_modules/nmo/lib/config.js:75:10)
ERR!     at /usr/local/lib/node_modules/nmo/lib/nmo.js:53:12
ERR!     at tryCatcher (/usr/local/lib/node_modules/nmo/node_modules/bluebird/js/main/util.js:24:31)
ERR!     at Promise._resolveFromResolver (/usr/local/lib/node_modules/nmo/node_modules/bluebird/js/main/promise.js:427:31)
ERR!     at new Promise (/usr/local/lib/node_modules/nmo/node_modules/bluebird/js/main/promise.js:53:37)
ERR!     at Object.load (/usr/local/lib/node_modules/nmo/lib/nmo.js:52:10)
ERR!     at Object.<anonymous> (/usr/local/lib/node_modules/nmo/bin/nmo-cli.js:17:5)
ERR!     at Module._compile (module.js:456:26)
ERR!     at Object.Module._extensions..js (module.js:474:10)
ERR! 
ERR! 
ERR! nmo: 1.0.0 node: v0.10.36
ERR! please open an issue including this log on https://github.com/robertkowalski/nmo/issues
```

**Comment by janl** [comment]
To be clear: any command produces this error.

**Comment by janl** [comment]
the latest git version seems to work

**Comment by janl** [comment]
This is on Mac, works fine on Linux with Node 0.12

**Comment by robertkowalski** [comment]
aw, did not fail on travis as travis also uses nvm for managing node installations - d'oh!

https://github.com/robertkowalski/nmo/pull/2 has a fix

**Comment by robertkowalski** [comment]
fix published as part of 1.0.1 - thanks for the detailed bug report for a very huge bug :)
